cmake/CmakeFunctionHelper.cmake

```diff
@@ -155,10 +155,10 @@ elseif(${TARGET} STREQUAL "AMD_GPU")  # need investigation
     endif()
   endif()
 else() # default cpu backend
-  set(gemm_configuration_0 "float"  64 "false" "false" "false" 64 8 8 8 8 1 1 "no_local" "naive" 8)
-  set(gemm_configuration_1 "float"  64 "false" "false" "false" 64 8 8 8 8 1 1 "no_local" "standard" 8)
-  set(gemm_configuration_2 "double" 64 "false" "false" "false" 64 8 8 8 8 1 1 "no_local" "naive" 8)
-  set(gemm_configuration_3 "double" 64 "false" "false" "false" 64 8 8 8 8 1 1 "no_local" "standard" 8)
+  set(gemm_configuration_0 "float"  64 "false" "false" "false" 64 8 8 8 8 1 1 "no_local" "naive" 1)
+  set(gemm_configuration_1 "float"  64 "false" "false" "false" 64 8 8 8 8 1 1 "no_local" "standard" 1)
+  set(gemm_configuration_2 "double" 64 "false" "false" "false" 64 8 8 8 8 1 1 "no_local" "naive" 1)
+  set(gemm_configuration_3 "double" 64 "false" "false" "false" 64 8 8 8 8 1 1 "no_local" "standard" 1)
```

Comment:
I don't quite understand how reducing the vector size improves vectorization

Reply:
The previous values of 8 for vector size were chosen somewhat arbitrarily before the no_local kernel was vectorized. However, upon testing the vectorization actually significantly reduces performance on Intel CPU (which uses these default CPU configurations) likely due to the increased register pressure so they have been set to 1 here (effectively disabling vectorization for default CPU).